docker-compose.yml

```diff
@@ -136,6 +144,20 @@ services:
       store:
         condition: service_healthy
 
+  ingest_loader:
```

Comment:
`ingest-loader` for consistency with the rest of the file?

Also, can you remove the first line (the version bit) from the docker-compose? It keeps complaining about this deprecation.